jobs/interfaces/searchengine.go

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package interfaces
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+type SearchEngineIndexerInterface interface {
```

Comment:
Where are we using this interface?

Reply:
In the enterprise code. And in the future in the Bleve code.